packages/codegen-ui-react/lib/__tests__/react-forms/__snapshots__/form-renderer-helper.test.ts.snap

```diff
@@ -70,7 +70,7 @@ exports[`form-render utils should generate before & complete types if datastore
 "{
     clearOnSuccess?: boolean;
     onSubmit?: (fields: mySampleFormInputValues) => mySampleFormInputValues;
-    onSuccess?: (fields: mySampleFormInputValues) => void;
+    onSuccess?: (result: any) => void;
```

Comment:
Not in love with the `any`. Don't we know the full shape of the object including Ids?